docs/source/extension/notebook.rst

```diff
@@ -146,11 +146,12 @@ How to extend the Notebook plugin
 
 We'll walk through two notebook extensions:
 
+-  adding a widget to the notebook header
 -  adding a button to the toolbar
 -  adding an ipywidgets extension
 
-Adding a button to the toolbar
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Adding a widget to the notebook header
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Very minor: I would flip them in order. I imagine more extension developers will be interested in adding a button rather than adding a widget.